numpy/core/_internal.py

```diff
@@ -839,7 +839,7 @@ def npy_ctypes_check(cls):
             # # (..., _ctypes._CData, object)
             ctype_base = cls.__mro__[-2]
         # right now, they're part of the _ctypes module
-        return 'ctypes' in ctype_base.__module__
+        return '_ctypes' in ctype_base.__module__
```

Comment:
Is this (still) necessary? I am not sure how it is related.

Reply:
it is only tangentially related, in that `ctypes` is not very specific and so can be confused, especially since we have a `numerictypes` module. I would like to leave the change even though it is not needed by this PR, since it may prevent someone in the future spending too much time staring at an error. Although `_ctypes` is only marginally better. This is all a hack around a cpython issue.

Reply:
Whoops, I hadnt considered numeri**ctypes""